docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -135,16 +163,37 @@ cluster, and the summarization part will affect the status summarizer.
 ## Design overview
 
 In very brief: the design is to reduce each edge placement problem to
-many instances of kcp's TMC problem.
+many instances of kcp's TMC problem.  Part of making many instances of
```

Comment:
Maybe "many instances of a single location/target TMC problem:" ?

Reply:
The latest push revises this wording.